theano/gpuarray/basic_ops.py

```diff
@@ -377,6 +390,17 @@ def f(*args):
     return f
 
 
+def get_dtype(s):
```

Comment:
There is many different dtype. Can you have a more precise name and add a docstring?